indico/modules/events/registration/util.py

```diff
@@ -997,6 +997,36 @@ def close_registration(regform):
         regform.start_dt = regform.end_dt
 
 
+@no_autoflush
+def clone_registration_form(regform):
```

Comment:
Instead of copying the logic from `RegistrationFormCloner`, can we extract it to a separate function and call it both here and in `RegistrationFormCloner`?

Reply:
I recommend adding a new `clone_single_regform` classmethod to the cloner, like we did in the ContributionCloner (`clone_single_contribution`).